src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -130,12 +135,18 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
             this.renderEntity(entityConf)
           )}
         </div>
+        ${this._config.footer
+          ? this.renderHeaderFooter(this._config.footer, "footer")
+          : ""}
       </ha-card>
     `;
   }
 
   static get styles(): CSSResult {
     return css`
+      :host {
+        --ha-card-border-radius: 20px;
```

Comment:
Is this an error? We don't want this on just one card

Reply:
oh shit that was my debugging :D 2px was too difficult to see if my patches for the heaer/footer class worked.